### PR TITLE
[EventDispatcher] Minor fixes

### PR DESCRIPTION
**Pull request by sebpacz**
I have added some minor fixes to the Event Dispatcher pages.

### CODE REVIEW

**Review by wouterj — APPROVED** [review]

File: event_dispatcher/before_after_filters.rst [hunk 1]
```diff
@@ -189,7 +189,7 @@ serve as a basic flag that this request underwent token authentication::
     {
         // ...
 
-        if ($controller[0] instanceof TokenAuthenticatedController) {
```
Comment (javiereguiluz) [inline event_dispatcher/before_after_filters.rst:192]: This change makes me a bit nervous ... but I never know the exact format of controllers (I know it's not always an array, but sometimes it is). Any thoughts? Thanks!

Comment (wouterj) [reply event_dispatcher/before_after_filters.rst:192]: In the previous example (line 137 of this file), array controllers are already taken care of. So this is a correct change imho

Comment (javiereguiluz) [reply event_dispatcher/before_after_filters.rst:192]: You are right! Sorry I missed this!

**Comment by javiereguiluz** [comment]
Nice fixes! Thanks Sebastian.

**Pull request merged**